03/styles/global.css

```diff
@@ -34,48 +35,46 @@ body {
     "text-area"
     "video-area"
     ;
+    max-width: 1400px;
 }
 
 .p {
     grid-area: picture-area;
-  
-    
 }
 
+img {
+    max-width: 100%;
+}
+
+
 .t {
     grid-area: text-area;
     
 }
 
 .v {
     grid-area: video-area;
+    max-height: 100%;
     
 }
 
 
 
-@media (min-width:599px){
-    body {
-        grid-template-areas: 
-        "picture-area"
-        "text-area"
-        "video-area"
-        ;
-    }
-}
 
 
-@media (min-width:1199px){
+
+
+@media (min-width:600px) and (max-width:1199px) {
```

Comment:
Jeśli definiujesz media w tym pliku to zrezygnowałbym z atrybutu `media` w pliku HTML gdzie dołączasz css.